--- bulldozer-0.2.3-orig/Makefile	2005-10-03 05:31:26.000000000 -0500
+++ bulldozer-0.2.3/Makefile	2006-01-05 15:58:04.000000000 -0600
@@ -10,7 +10,7 @@
 
 # =============================================================================
 
-prefix = /usr/local
+prefix = /usr
 datadir = $(prefix)/share
 
 libdir = $(shell pkg-config --variable=libdir libnautilus-extension)
@@ -37,16 +37,16 @@
 CFLAGS = \
 	$(shell $(PKG_CONFIG) --cflags libnautilus-extension libxml-2.0) \
 	-DIMAGEDIR='"$(imgdir)"' -DLOCALEDIR='"$(locdir)"' \
-	-DGETTEXT_PACKAGE='"$(PACKAGE)"' -Wall -fPIC -std=c99
+	-DGETTEXT_PACKAGE='"$(PACKAGE)"' -Wall -std=c99 -O2 -pipe
 LDLIBS = \
 	$(shell $(PKG_CONFIG) --libs libnautilus-extension libxml-2.0)
 LDFLAGS = \
-	-shared
+	-shared -Wl,--enable-auto-image-base
 
 # =============================================================================
 
 LIBRARY = \
-	$(CURDIR)/libnautilus-$(PACKAGE)-extension.so
+	$(CURDIR)/cygnautilus-$(PACKAGE)-extension.dll
 SOURCES = \
 	$(srcdir)/bulldozer-ant-script.c \
 	$(srcdir)/bulldozer-extension.c \
